### ISSUE TITLE
Cannot verify in the selected network

### ISSUE THREAD

**Issue by alexkeston** [Machinalabs/remix-etherscan-plugin#4]
Hi, whenever I tried to verify my smart contract in etherscan, the error is coming up called "Cannot verify in the selected network". Please kindly help!

**Comment by EdsonAlcala** [comment]
hey sorry ! Just saw this, gonna take a look

**Comment by alexkeston** [comment]
Hey,

I planned to create an erc 20 token But, I changed my mind when I saw
factors like unstability, dependibility, sovereignty, centralisation and
gas fees.

I know there are so many pros of creating erc20 token but, Now I have
turned my mind to creating our own blockchain.

Questions I have:

Can we create our own blockchain using Ethereum?

And

Can we have 100% control over updating and editing the blockchain.

About Coin "STITHULF" : So, I want to create a coin with specific total
supply which solves problem of inflation. Also, I have planned to create
this currency as internet's currency which will be used to buy any e
commerce product.

I would be releasing white paper next week where I will be explaining the
total plan and revolution we are creating.

Thanks, your help will be so important for my project.
